src/bind.rs

```diff
@@ -485,81 +420,17 @@ where
                 *next = Some(svc);
                 ready
             }
-        };
-
-        // If there was a connect error, don't terminate this BoundService
-        // completely. Instead, simply clean up the inner service, prepare to
-        // make a new one, and tell our caller that we could maybe be ready
-        // if they call `poll_ready` again.
-        //
-        // If they *don't* call `poll_ready` again, that's ok, we won't ever
-        // try to connect again.
```

Comment:
It never made sense to me that this code was in the `NormalizeUri` service --- moving it to the new reconnect service is great.